client/src/App.js

```diff
@@ -8,11 +9,14 @@ import VideoPlayer from "./Components/VideoPlayer";
 function App() {
   const [cards, setCards] = useState([]);
   const [open, setOpen] = useState(false);
```

Comment:
Please give it some more meaningful name, instead of having just "open", not really clear what should be opened :)